packages/react/stories/Shortcut.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import { select, text, boolean } from "@storybook/addon-knobs/react";
+import Shortcut from "elements/components/Shortcut/index";
+
+const icons = [
+  "add",
```

Comment:
Do these exist as data anywhere?

Reply:
hmmm good question. will check

Reply:
They don't exist as data. Don't let that block this PR. Would be nice to get that later.